openra/templates/header.html

```diff
@@ -5,7 +5,7 @@
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 	<meta name="viewport" content="width=1024" />
-	<title>OpenRA Resource Center{{ title }}</title>
+    <title>OpenRA Resource Center{% if title %} - {{ title }}{% endif %}</title>
```

Comment:
The indentation looks off here.

Reply:
I guess it's a tab / space mix. I'll take a look

Reply:
Ok these all use tabs now instead of spaces (as that seems to be the convention here)